src/maintenance/repositories/official/installing.md

```diff
@@ -0,0 +1,12 @@
+# Installing Repositories
+
+Void provides official [sub-repositories](./sub-repositories/index.md) that may
```

Comment:
I think it was discussed that they aren't sub- anything.  Please change this language.

Reply:
Missed this one. Oops.